pkgs/games/aaaaxy/default.nix

```diff
@@ -0,0 +1,92 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, alsa-lib
+, libglvnd
+, libX11
+, libXcursor
+, libXext
+, libXi
+, libXinerama
+, libXrandr
+, libXxf86vm
+, go-licenses
+, pkg-config
+#, xvfb-run
+}:
+
+buildGoModule rec {
+  pname = "aaaaxy";
+  version = "1.3.372";
+
+  src = fetchFromGitHub {
+    owner = "divVerent";
+    repo = pname;
+    rev = "v${version}";
+    hash = "sha256-vm3wA8lzoaJ5iGwf2nZ0EvoSATHGftQ77lwdEjGe2RU=";
+    fetchSubmodules = true;
+  };
+
+  vendorHash = "sha256-WEK7j7FMiue0Fl1R+To5GKwvM03pjc1nKig/wePEAAY=";
+
+  buildInputs = [
+    alsa-lib
+    libglvnd
+    libX11 libXcursor libXext libXi libXinerama libXrandr
+    libXxf86vm
+  ];
+
+  nativeBuildInputs = [
+    go-licenses
```

Comment:
Please patch out the requirement for this

Reply:
I don't want to patch it out. The in-game credits contain a list of software licenses. If I patch out everything that requires the `go-licenses` program, the credits will not show the licenses anymore. It feels wrong to modify the game in this way.

Reply:
That sounds actually useful. I first thought this runs a license check on the software.